packages/components/psammead-story-promo/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bbc/psammead-story-promo",
-  "version": "4.0.0-alpha.5",
+  "version": "4.0.0-alpha.6",
```

Comment:
I think `@bbc/psammead-timestamp-container` should be added as a `devDependency` since we are using it now in the stories and it is not in the root `package.json`.

Reply:
We don't need this now :)